docs/howto/graph-api/misc.md

```diff
@@ -0,0 +1,319 @@
+
+```json hl_lines="11-19"
+{
+   "search":{
+      "m_request":{
+         "query":{
+            "owner":"all",
+            "query":{
+               "upload_create_time":{ "gt":"2025-01-01" }
+            }
+         }
+      },
+      "*":{
+         "entry":{
+            "entry_create_time":{
+               "m_request":{
+                  "directive":"plain"
+               }
+            }
+         }
+      }
+   }
+}
+```
+
+The response will look like the following.
+Note when a `query` field is provided, it will be returned in the `m_response` field.
+Also, the `pagination` field will always be returned, even if not specified in the request.
+
+```json
+{
+   "search":{
+      "m_response":{
+         "include":[ "*" ],
+         "query":{
+            "owner":"all",
+            "query":{
+               "upload_create_time":{ "gt":"2025-01-01" }
+            },
+            "aggregations":{}
+         },
+         "pagination":{
+            "page_size":10,
+            "order_by":"entry_id",
+            "order":"asc",
+            "page_after_value":null,
+            "page":1,
+            "page_offset":null,
+            "total":37,
+            "next_page_after_value":"HcLSOzI89sGMCgYnhhfztI4Z1qxY",
+            "page_url":null,
+            "next_page_url":null,
+            "prev_page_url":null,
+            "first_page_url":null
+         }
+      },
+      "-L073PFe_PxW90kci4UwxMgUO20O":{ "entry":{ "entry_create_time":"2025-06-23T16:34:50.684000" } },
+      "3LGsPbtrFaiUHp_pG-j9WxLa24Gc":{ "entry":{ "entry_create_time":"2025-06-23T16:07:56.530000" } },
+      "6EMY3osB1O7izLHlh1dXtoZYn0Ms":{ "entry":{ "entry_create_time":"2025-06-23T15:37:17.793000" } },
+      "75dfpDPLnWz9k35aUBLuUb0StWz9":{ "entry":{ "entry_create_time":"2025-06-23T16:34:50.287000" } },
+      "7YRYJDKWSNCTK5CRsBI-oopL0iep":{ "entry":{ "entry_create_time":"2025-06-23T16:07:56.256000" } },
+      "8eYh9XB_vbKHMK8m_N7DK1U0dBlC":{ "entry":{ "entry_create_time":"2025-06-23T16:34:50.231000" } },
+      "9OpqgAwjqWzr0URSnsRywqm1S52Q":{ "entry":{ "entry_create_time":"2025-06-23T16:07:56.586000" } },
+      "AV12t0vir9EyCHNrmnXTlpROy5rz":{ "entry":{ "entry_create_time":"2025-06-23T16:34:50.060000" } },
+      "FeuWgNCZ_YCevhwQyR-k5RJ1_uuY":{ "entry":{ "entry_create_time":"2025-06-18T01:23:16.343000" } },
+      "HcLSOzI89sGMCgYnhhfztI4Z1qxY":{ "entry":{ "entry_create_time":"2025-06-23T16:07:56.364000" } }
+   }
+}
+```
+
+??? note "data duplication"
+    Some fields are both stored in the `MongoDB` database and the `Elasticsearch` index.
+    And it is likely that the desired information can be directly fetched from `Elasticsearch`.
+    In this case, there is no need to navigate to the `MongoDB` database.
+    However, if the request needs to access archive, it has to use `entry -> archive` path if starting from `search`.
```

Comment:
This is an important point. Just for me to know: does the API already skip MongoDB if the requested information is in Elasticsearch?

Reply:
No, it returns a list of "entry metadata", depending on the query and required, it may or may not include metadata stored in mongo.
You can still navigate to the corresponding mongo entry metadata via `entry` token.
But mostly like you can directly get them by assigning `required` field in the search query.